When adding a contact from recent calls, it goes to home screen and throws a database error.

1. Open phone
2. Open Phone App
3. Navigate to contacts and dial a number.
4. End the call.
5. Go to recent calls and right click on new contact. (Same occurs with key control.)
6. Phone will go to home screen.

This results in this error:
```
[ERROR] [MySQL] [gcphone] An error happens on MySQL for query "INSERT INTO phone_users_contacts (`identifier`, `number`,`display`) VALUES('steam:110000112964a88', @number, @display)": ER_BAD_NULL_ERROR: Column 'display' cannot be null
````

You can also achieve this by trying to add to contacts an existing contact.

--- a/src_htmlPhone/src/components/Appels/AppelsRecents.vue
+++ b/src_htmlPhone/src/components/Appels/AppelsRecents.vue
@@ -55,7 +55,7 @@ export default {
     }
   },
   methods: {
-    ...mapActions(['startCall', 'appelsDeleteHistorique', 'appelsDeleteAllHistorique', 'addContact']),
+    ...mapActions(['startCall', 'appelsDeleteHistorique', 'appelsDeleteAllHistorique']),
     getContact (num) {
       const find = this.contacts.find(e => e.number === num)
       return find

--- a/src_htmlPhone/src/components/Appels/AppelsRecents.vue
+++ b/src_htmlPhone/src/components/Appels/AppelsRecents.vue
@@ -110,13 +110,16 @@ export default {
     },
     save (numero) {
       if (this.id !== -1) {
-        this.addContact({
-          number: numero
+        this.$router.push({
+          name: 'contacts.view',
+          params: {
+            id: 0,
+            number: numero
+          }
         })
       } else {
         console.log('No añadido')
       }
-      history.back()
     },
     stylePuce (data) {
       data = data || {}

